components/tools/OmeroPy/src/omero/plugins/ldap.py

```diff
@@ -163,7 +166,15 @@ def getdn(self, args):
         if dn is not None and dn.strip():
             self.ctx.out(dn)
 
-    @admin_only
+    @admin_only(AdminPrivilegeModifyGroup)
+    def update_group(self, iupdate, group):
+        iupdate.saveObject(group)
+
+    @admin_only(AdminPrivilegeModifyUser)
+    def update_user(self, iupdate, user):
+        iupdate.saveObject(user)
+
+    @admin_only()
```

Comment:
This PR should improve things from where they were without this PR.

Still, outwith this PR but in the context of that distinction/confusion it should thus be noted that I may have missed that some of the remaining `@admin_only()` may still benefit from specific privileges being required. More severely, one may worry about that a light admin running a CLI subcommand may cause,

1. First step of subcommand happens.
1. Second step of subcommand fails (needs different privilege).
1. No useful rollback of the first, as previously it seemed implausible that the first would succeed yet not the second.

I didn't find such in my CLI review but the possibility ought still be considered.